### ISSUE TITLE
(debian Buster clean inst): Phoenix run file doesnt contain fqdn as specified in /etc/firezone/firezone.rb.

### ISSUE THREAD

**Issue by frankuit** [CloudFire-LLC/cloudfire#263]
Nice tool, great potential imo :)

I've done a test-installation on debian buster backported kernel 5.10.
modified the /etc/firezone/firezone.rb file with the fqdn name: default['firezone']['fqdn'] = 'myservername.domainname.com' <changed for privacy ofcourse.
other then that, i've only added our certificate filenames.

then, firezone-ctl reconfigure, and created initial admin.

The webinterface lets me login. however, pressing a button such as "Add device" doesn't seem to work.
I've Located the problem via the logfile, stating that there is a discrepancy betweeon the hostname used and the config value.
saying:
==> /var/log/firezone/phoenix/current <==
2021-09-30_11:57:27.41499 13:57:27.414 [error] Could not check origin for Phoenix.Socket transport.

so in /opt/firezone/sv/phoenix/run
export URL_HOST="myservername" 
it's missing the .domainname.com
after adding that, and restarting with firezone-ctl restart, it works, and i can click buttons in the webinterface.

extra info:

/etc/hostname contains: myservername (without the domainname.com)
/etc/hosts contains 127.0.0.1    myservername     localhost (also without domainname.com)

Hope this helps, 
Cheers!
Frank

**Comment by jamilbk** [comment]
Hi Frank,

Thanks for the bug report. I suspect this is a bug with how we're applying changes to the `default['firezone']['fqdn']` config var to the `default['firezone']['url_host']` config var.

Could you make sure `default['firezone']['url_host']` is set to `myservername.domainname.com` explicitly in `/etc/firezone/firezone.rb` and try that? You should see it a few lines down:

```ruby
# The fully qualified domain name. Will use the node's fqdn if nothing is
# specified.
default['firezone']['fqdn'] = (node['fqdn'] || node['hostname']).downcase
...
# Used for generating URLs that point back to this application.
default['firezone']['url_host'] = node['firezone']['fqdn']
...
```



I'll work on a fix for this.

**Comment by iAmateur** [comment]
Thank you for your work! I too can confirm the same issue and solution with Ubuntu 20.04.3, with the correct fqdn setting set in /etc/firezone/firezone.rb

**Comment by jamilbk** [comment]
This is fixed in the [0.1.9 release](https://github.com/firezone/firezone)

**Comment by iAmateur** [comment]
Again, thank you jamilbk, for your work. Can confirm 0.1.9 is working; I deployed your uninstall script, reinstalled 0.1.9 and success! Cheers
